.github/workflows/unit-tests-framework.yml

```
@@ -167,6 +164,16 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Ensure cache directory exists
+        run: mkdir -p /root/.cache/bionemo
+
+      - name: Cache bionemo test data
+        uses: actions/cache@v4
+        with:
+          path: /root/.cache/bionemo
+          key: bionemo-test-data-v1  # bump when data format changes
+          retention-days: 14
```

actionlint (1.7.7)

input "retention-days" is not defined in action "actions/cache@v4". available inputs are "enableCrossOsArchive", "fail-on-cache-miss", "key", "lookup-only", "path", "restore-keys", "save-always", "upload-chunk-size"

(action)